exporter/lokiexporter/README.md

```diff
@@ -0,0 +1,62 @@
+# Loki Exporter
+
+Exports data via HTTP to [Loki](https://grafana.com/docs/loki/latest/). By default, this exporter requires TLS and
+offers queued retry capabilities.
+
+Supported pipeline types: logs
+
+## Getting Started
+
+The following settings are required:
+
+- `endpoint` (no default): The target URL to send Loki log streams to (
+  e.g.: https://loki.example.com:3100/loki/api/v1/push).
+
+
+- `attributes_for_labels` (no default): List of allowed attributes to be added as labels to Loki log 
+  streams. This is a safety net to help prevent accidentally adding dynamic labels that may significantly increase 
+  cardinality, thus having a performance impact on your Loki instance. See the 
+  [Loki label best practices](https://grafana.com/docs/loki/latest/best-practices/current-best-practices/) page for 
+  additional details on the types of labels you may want to associate with log streams.
+
+The following settings can be optionally configured:
+
+- `insecure` (default = false): When set to true disables verifying the server's certificate chain and host name. The
+  connection is still encrypted but server identity is not verified.
+- `ca_file` (no default) Path to the CA cert to verify the server being connected to.
+- `cert_file` (no default) Path to the TLS cert to use for client connections when TLS client auth is required. 
+  Should only be used if `insecure`
```

Comment:
Since these are for client auth, I guess they're independent of the `insecure` flag? `insecure` is to allow TLS even if `ca_file` isn't set I think

Reply:
That is correct. The `insecure` flag configures the HTTP client to not validate the server certificate, so it wouldn't make sense to provide a certs if validation is disabled. 

Updating docs so that it says "Should only be used if `insecure` is set to false.